static/js/app.js

```diff
@@ -823,37 +787,153 @@ $(function () {
                     scaledSize: new google.maps.Size(30, 50)
                 };
             }
-            
             return image_url;
+        },
+        load_filter: function() {
+            // TODO: Change to switch which will only go through requested filters
```

Comment:
I think it is.
I'll remove the TODO and open an issue which will improve `loadFiler()`